src/components/Workshops/Workshop.jsx

```diff
@@ -20,12 +22,29 @@ export default function Workshop({ title, youtube, author, description, readme,
                     </button> : null;
     return (
         <div className='workshop-container'>
-            <ReactPlayer 
-                className='workshop-video'
-                id={'id_' + title.replace(/ /g, '_')}
-                controls={true}
-                width='100%'
-                url={youtube} />
+
+            {/* Lazy loads thumbnail images
+            video player is not actually loaded until a thumbnail is clicked */}
+            {!isPlayerVisible && (
+                <LazyLoadImage
+                    className='video-thumbnail'
+                    src={`https://img.youtube.com/vi/${youtube.split('/').pop()}/0.jpg`}
+                    alt={title}
+                    effect='blur'
+                    onClick={() => setIsPlayerVisible(true)}
+                    style={{ cursor: 'pointer', width: '100%' }}
+                />
+            )}
+            {isPlayerVisible && (
+                <ReactPlayer
+                    className='workshop-video'
+                    id={'id_' + title.replace(/ /g, '_')}
+                    controls={true}
+                    width='100%'
+                    url={youtube}
+                />
+            )}

```

Comment:
Let's update this-- as Jenna commented earlier, the lazy loading is great but the UX somewhat suffers. Looking at the deploy, I think it's hard to tell that it's a youtube video you can click on.

It'd be great if we could lazily load the current youtube embed that we have so it's still easily identifiable as a video.
<img width="427" alt="image" src="https://github.com/user-attachments/assets/46ca91cb-4f92-4c4a-ae40-bade8fde6111" />